src/main/java/codeu/model/store/basic/ProfileStoreTest.java

```diff
@@ -0,0 +1,92 @@
+package codeu.model.store.basic;
+
+import codeu.model.data.Profile;
+import codeu.model.store.persistence.PersistentStorageAgent;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito; 
+
+public class ProfileStoreTest {
+
+  private ProfileStore profileStore;
+  private PersistentStorageAgent mockPersistentStorageAgent;
+
+  private final Profile PROFILE_ONE =
```

Comment:
The current Profile class doesn't have a field related to time. Was this a change that you made?